integration/otlp_ingestion_test.go

```diff
@@ -72,4 +74,20 @@ func TestOTLPIngestion(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, model.ValMatrix, rangeResult.Type())
 	require.Equal(t, expectedMatrix, rangeResult.(model.Matrix))
+
+	// Push series with histograms to Mimir
+	series = []prompb.TimeSeries{
+		{
+			Labels: []prompb.Label{
+				{Name: "foo", Value: "bar"},
+			},
+			Histograms: []prompb.Histogram{
+				remote.HistogramToHistogramProto(1337, test.GenerateTestHistograms(1)[0]),
+			},
+		},
+	}
+	_, err = c.PushOTLP(series)
+	require.NoError(t, err)
+	// TODO query and assert series that contain native histogram once querying
```

Comment:
Both ingestion and querying are now supported in `main`. To ingest, one needs to set `-ingester.native-histograms-ingestion-enabled=true` flag. Querying requires `-query-frontend.query-result-response-format=protobuf` flag.

Can we update this integration test to make sure it histograms work as expected?

Reply:
done

Reply:
`-ingester.native-histograms-ingestion-enabled=true` is set in `NewSingleBinary`